changelog.md

```diff
@@ -17,6 +17,8 @@
   backends, and it is compatible with Python's behavior,
   e.g. `formatFloat(3.14159, precision = 0)` is now `3`, not `3.`.
 - Global variable `lc` has been removed from sugar.nim.
+- `distincBase` has been moved from sugar.nim to typetraits and now implemented as
```

Comment:
Minor, typo: `distincBase` should be `distinctBase` (added `t`).

(on the following line too)